packages/jumpstarter-driver-corellium/jumpstarter_driver_corellium/corellium/api_test.py

```
@@ -0,0 +1,183 @@
+import os
+
+import pytest
+import requests_mock
```

Ruff (0.8.2)

`requests_mock` imported but unused

Remove unused import: `requests_mock`

(F401)